### ISSUE TITLE
Workbench sliceviewer ellipsoid background shell not plotted correctly

### ISSUE THREAD

**Issue by RichardWaiteSTFC** [mantidproject/mantid#29087]
The inner radius of the ellipsoid background shell is not plotted correctly in workbench sliceviewer. A comparison of mantidplot (left) and workbench (right) is shown below

![image](https://user-images.githubusercontent.com/55979119/88943579-90cb1b00-d283-11ea-8f43-a0f757bd8d18.png)

MantidPlot is displaying an background inner radius very similar to the peak radius as it should be, but the workbench sliceviewer displays a much larger inner radius.

The spherical peakshape is plotted correctly in workbench.

### Platforms affected
workbench

**Comment by martyngigg** [comment]
@RichardWaiteSTFC Would you have a script that could reproduce this?

**Comment by RichardWaiteSTFC** [comment]
@martyngigg 
Note that the background inner radius is the same as the peak radius - but not on the sliceviewer plot
```
Load(Filename='SXD23767.raw', OutputWorkspace='SXD23767', LoaderName='LoadRaw', LoaderVersion='3')
ConvertToDiffractionMDWorkspace(InputWorkspace='SXD23767', OutputWorkspace='SXD23767MD', OneEventPerBin='0')
FindSXPeaks(InputWorkspace='SXD23767', PeakFindingStrategy='AllPeaks', AbsoluteBackground=2000, 
    ResolutionStrategy='AbsoluteResolution', XResolution=200, PhiResolution=3, TwoThetaResolution=3, 
    OutputWorkspace='peaks')
IntegratePeaksMD(InputWorkspace='SXD23767MD', PeakRadius=0.35, BackgroundInnerRadius=0.35,
    BackgroundOuterRadius=0.5, Ellipsoid=True, PeaksWorkspace='peaks', OutputWorkspace='peaks_int', 
    UseOnePercentBackgroundCorrection=False)
```

**Comment by RichardWaiteSTFC** [comment]
Fixed by PR #30403
